src/fparser/two/tests/test_fortran2003.py

```diff
@@ -2964,6 +2964,9 @@ def test_procedure_stmt():  # R1206
     assert isinstance(obj, tcls), repr(obj)
     assert str(obj) == "MODULE PROCEDURE a, b"
 
+    with pytest.raises(NoMatchError):
```

Comment:
Please add a comment to say that the double colon is only valid in Fortran2008 onwards.

Reply:
Good idea.